core/src/main/java/org/jdbi/v3/mapper/CollectorFactory.java

```diff
@@ -11,7 +11,7 @@
  * See the License for the specific language governing permissions and
  * limitations under the License.
  */
-package org.jdbi.v3.tweak;
+package org.jdbi.v3.mapper;
```

Comment:
Should the collector stuff go into its own package? Seems odd to put it in `mapper`.